libraries/botframework-connector/tests/auth.test.js

```diff
@@ -5,14 +5,24 @@ const { StatusCodes } = require('botframework-schema');
 
 describe('Bot Framework Connector - Auth Tests', function() {
 
-    describe('Connector Tokens', function() {
+    describe('Connector Tokens', async function() {
```

Comment:
We should remove the `async` keyword, so there is no room for calling an `await` keyword inside the describe callback funcion scope.